public/manifest.webapp

```diff
@@ -1,21 +1,23 @@
 {
-    "version": "0.1",
-    "name": "Capture",
-    "description": "Capture App",
-    "launch_path": "index.html",
-    "icons": {
-        "16": "/assets/icons/track16.png",
-        "48": "/assets/icons/track48.png",
-        "128": "/assets/icons/track128.png"
-    },
-    "developer": {
-        "name": "HISP",
-        "url": "http://dhis2.org"
-    },
-    "default_locale": "en",
-    "activities": {
-        "dhis": {
-            "href": "../../.."
```

Comment:
When the app is uploaded to play the path will be something like https://play.dhis2.org/2.29/api/apps/Capture/index.html#/. So I think we need the "../../.."

Reply:
Right, thanks. I'll fix it.

Reply:
Fixed.